APE1.rst

```diff
@@ -139,6 +139,28 @@ An APE's status can
   rationales, so that new community members can understand why a decision was
   made.
 
+Modifications to an APE
+.......................
+
+**Addendum: July 22, 2020**
+
+An APE can be modified, with one person acting as "champion" for the
+APE update. Similar to the APE development procedure, the update
+champion should discuss their desired changes with the astropy
+community (e.g., through the astropy-dev mailing list).
+
+Following a discussion on astropy-dev, the APE update shall be
+submitted as a Pull Request. The submission of the Pull Request should
```

Comment:
```suggestion
submitted as a pull request. The submission of the pull request should
```

Reply:
I capitalized because it was previously capitalized in this document, but I don't really care!

Reply:
Oh interesting.  I think github changed their convention on this - it used to be capitalized, but no longer is (https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests).  So :+1: to @bsipocz 's suggestion

Reply:
Fixed